src/routes/github/github-oauth-router.ts

```diff
@@ -8,17 +8,20 @@ import { Tracer } from "config/tracer";
 import { envVars } from "config/env";
 import { GithubAPI } from "config/github-api";
 import { Errors } from "config/errors";
-import { getGitHubHostname, getGitHubApiUrl } from "~/src/util/get-github-client-config";
+import { getGitHubApiUrl } from "~/src/util/get-github-client-config";
 import { createHashWithSharedSecret } from "utils/encryption";
 
 const logger = getLogger("github-oauth");
-
-const githubClient = envVars.GITHUB_CLIENT_ID;
-const githubSecret = envVars.GITHUB_CLIENT_SECRET;
 const baseURL = envVars.APP_URL;
 const scopes = ["user", "repo"];
 const callbackPath = "/callback";
 
+const getRedirectUrl = async (req, res, state) => {
```

Comment:
Is the "support GHE in redirect callback" feature covered in any tests? If so, ignore this comment. Otherwise, it'd be great if you can add some?

Reply:
Test case added for testing the redirectUrl in the `GithubOAuthRouter`